server/models/Profile.js

```diff
@@ -0,0 +1,61 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  user_id: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+  },
+  firstname: {
+    type: String,
+  },
+  lastname: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  gender: {
+    type: String,
+  },
+  birthdate: {
+    type: Date,
+  },
+  phone_number: {
+    type: String,
+  },
+  photo_url: {
+    type: String,
+  },
+  location: {
+    city: {
+      type: String,
+      required: true,
+    },
+    province: {
+      type: String,
+      required: true,
+    },
+    country: {
+      type: String,
+      required: true,
+    },
+    index: true,
+    required: true,
+  },
+  availability: [
+    {
+      start_at: {
+        type: Date,
+        required: true,
+        index: true,
+      },
+      end_at: {
+        type: Date,
+        required: true,
+      },
+    },
+  ],
+  hourlyWage: {
```

Comment:
This is subtle but I don't think `wage` is the best choice of word for this. Maybe `hourlyRate` or `hourlyFee` instead? What do you think?

Reply:
Yeah that was confusing name. Will fix it.